documents/affiliated_package_review_process.md

```diff
@@ -0,0 +1,130 @@
+which takes you directly to an editor. If you get a message saying **You need
+to fork this repository to propose changes** then click on the green button
+to confirm the forking:
+
+<p align="center">
+  <img src="images/propose_fork.png" alt="fork" width="600">
+</p>
+
+Copy the following template entry and fill out the
+details:
+
+    {
+      "name": "Your package name",
+      "maintainer": "Your name <your email address>",
+      "stable": true,
+      "home_url": "URL to the home page or documentation",
+      "repo_url": "URL to the repository",
+      "pypi_name": "PyPI name if it exists, set to an empty string otherwise",
+      "description": "A few sentence description of your package",
+      "review": {
+         "functionality": "To be filled out by the reviewer",
+         "ecointegration": "To be filled out by the reviewer",
+         "documentation": "To be filled out by the reviewer",
+         "testing": "To be filled out by the reviewer",
+         "devstatus": "To be filled out by the reviewer",
+         "python3": "To be filled out by the reviewer",
+         "last-updated": "To be filled out by the reviewer"
+      }
+    },
+
+then add it at the top of the ``registry.json`` file, starting on line 3,
+after:
+
+    {
+      "packages": [
+
+Once this is done, under **Propose file change** you can enter a short
+description for the commit then click on the green **Propose file change**
+button:
+
+<p align="center">
+  <img src="images/propose_commit.png" alt="commit" width="650">
+</p>
+
+On the next screen, check the changes you made, and click on the green
+**Create pull request** button:
+
+<p align="center">
+  <img src="images/propose_create_1.png" alt="create_1" width="600">
+</p>
+
+Finally, give a title to the pull request such as **Proposed affiliated
+package: your package name** and include a short description of the
+package, then click on **Create pull request**:
+
+<p align="center">
+  <img src="images/propose_create_2.png" alt="create_2" width="600">
+</p>
+
+Once this is done, send an email to https://groups.google.com/forum/#!forum/astropy-dev
+(which is the main developer list) to introduce your package and say that
+you would like it to be considered as an affiliated package. Make sure you
+include a link to the pull request you opened above. Once you've done this,
+you're all set! The next steps are the review, for which the results will be
+posted on the pull request and also in a reply to your email.
+
+## Reviewing affiliated packages
+
+*These instructions are intended for coordinators and reviewers*
+
+One of the coordination committee members will be in charge of the review
+process (not necessarily the same person for each package). This coordination
+committee member (hereafter the *coordinator* will find someone to carry out the
+main review of the package (hereafter the *reviewer*). The pool of available
+reviewers will be anyone who has an official 'role' on [this
```

Comment:
Change link text to `the Astropy team` (I've lately gotten down on link text like "this page" and I find that one can almost always find something more descriptive).  

The single quotes on `role` is not really needed.